packages/genesys-spark-components/src/components/beta/gux-rich-text-editor/gux-rich-text-editor-beta.tsx

```diff
@@ -288,12 +298,25 @@ export class GuxRichTextEditor {
     return this.renderSlot('global-action', 'gux-global-action-container');
   }
 
+  private hasToolbarChildren(): boolean {
+    const children = [
+      this.renderTypographicalEmphasis(),
+      this.renderTextStyling(),
+      this.renderListsIndentation(),
+      this.renderInserting(),
+      this.renderTextEditorMenu(),
+      this.renderGlobalAction()
+    ].filter(child => child !== null && child !== undefined);
+    return children.length > 0;
```

Comment:
This is fine but you could also use [Array.some](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some) here.

```
return [
      this.renderTypographicalEmphasis(),
      this.renderTextStyling(),
      this.renderListsIndentation(),
      this.renderInserting(),
      this.renderTextEditorMenu(),
      this.renderGlobalAction()
    ].some(child => child !== null && child !== undefined);
```